apps/podium-service/chalicelib/repository/repository.py

```diff
@@ -19,6 +19,28 @@
 #     return True
 
 
+def add_projects(projects: list[data_types.Projects]) -> Optional[str]:
+    def add(session: Session) -> str:
+        to_insert = []
+        verticals = get_verticals()
+        for project in projects:
+            to_insert.append(
+                models.Project(
+                    vertical_id=verticals[verticals.index(project.vertical)],
+                    name=project.name,
+                    team=project.teamMembers,
+                    description=project.description,
+                    image_url=project.imageUrl,
+                    devpost_url=project.devpostUrl,
+                    video_url=project.videoUrl,
+                )
+            )
+        session.add_all(to_insert)
+
+    projects_response = run_transaction(sessionmaker(engine), add)
```

Comment:
Currently your add function doesn't return anything so projects_response will be None. Either return nothing from this function or return the list of added project IDs